packages/expo-in-app-purchases/android/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
So far we deliberately didn't make unimodules' projects a full-fledged Android projects with `gradlew` and all the Gradle commands were always done through `android`. Does it make sense to make this module an exception, or is it just a leftover of some initial change? (Since this module also doesn't have `gradle/wrapper/`, so this script shouldn't work if I understand it correctly? 🤔 )

Reply:
Yeah I wasn't sure why this was included. Should it be deleted?